docs/developer-docs/latest/setup-deployment-guides/installation/docker.md

```diff
@@ -101,6 +101,7 @@ version: '3'
 services:
   strapi:
     image: strapi/strapi
+    command: bash -c "npm install && yarn start"
```

Comment:
I would recommend not using both tools, as npm creates `package-lock.json` files while yarn creates `yarn.lock` and this could cause issues.

So, until [npm supports yarn.lock files](https://github.blog/2021-02-02-npm-7-is-now-generally-available/#changes-to-the-lockfile) (which is a matter of months, apparently), I would go either with npm or yarn but use only one.

So please update this line:
* either with `npm install && npm run start`
* or with `yarn && yarn start`

Thanks!